packages/autorest.typescript/package.json

```diff
@@ -36,7 +36,7 @@
     "validate-version-tolerance": "npm run start-test-server:v2 & npm run generate-version-tolerance && npm run generate-version-tolerance:tests && npm run test-version-tolerance && npm run stop-test-server",
     "generate-version-tolerance": "ts-node ./test/commands/version-tolerance.ts",
     "generate-version-tolerance:tests": "ts-node ./test/commands/version-tolerance.ts --test-gen",
-    "regen-test-baselines": "npm run generate-swaggers && npm run rlc-generate-swaggers && npm run smoke-test",
+    "regen-test-baselines": "npm run generate-swaggers && npm run rlc-generate-swaggers && npm run generate-version-tolerance && npm run smoke-test",
```

Comment:
I am not sure why we remove the generate-version-tolerance for this case?

Reply:
This is manually edited.  It probably was never added to the `regen-test-baselines` script.  I found some baselines were not re-generated thus CI failed.  So I am adding it.

Reply:
oh okay, thanks.